packages/roosterjs-editor-dom/lib/utils/applyFormat.ts

```diff
@@ -8,7 +8,7 @@ import { DefaultFormat } from 'roosterjs-editor-types';
 export default function applyFormat(element: HTMLElement, format: DefaultFormat) {
     if (format) {
         let elementStyle = element.style;
-        let { fontFamily, fontSize, textColor, backgroundColor, bold, italic, underline } = format;
+        let { fontFamily, fontSize, textColor, backgroundColor, bold, italic, underline, originalSourceBackgroundColor, originalSourceTextColor } = format;

```

Comment:
do originalSourceBackgroundColor / originalSourceTextColor still exist on DefaultFormat? should they? since we reworked it to be a getter.